cli/src/main.rs

```diff
@@ -45,6 +45,10 @@ struct Args {
     #[arg(short, long, default_value = "false", requires = "stdin")]
     rustfmt: bool,
 
+    /// Macros to be formatted.
+    #[arg(long, num_args=1.., value_delimiter= ' ')]
+    format_macros: Option<Vec<String>>,
```

Comment:
I wonder if including `override` in the name of the setting would make sense...
`override_macro_names`? Not sure, what do you think?

Reply:
Much better!